doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -119,45 +119,45 @@ analysis on inductive or co-inductive objects (see :ref:`variants`).
 
 .. tacn:: destruct {+, @induction_clause } {? @induction_principle }
 
-   .. insertprodn induction_clause induction_principle
+   .. insertprodn induction_clause induction_arg
 
    .. prodn::
       induction_clause ::= @induction_arg {? as @or_and_intropattern } {? eqn : @naming_intropattern } {? @occurrences }
-      induction_arg ::= @natural
-      | @one_term_with_bindings
-      induction_principle ::= using @one_term {? with @bindings } {? @occurrences }
+      induction_arg ::= @one_term_with_bindings
+      | @natural
 
    Performs case analysis by generating a subgoal for each constructor of the
-   inductive or co-inductive type in the :n:`@one_term` in :n:`@induction_arg`.
-   In each new subgoal, the tactic replaces :n:`@one_term` with the associated
-   constructor (applied to its arguments).   Unlike :tacn:`induction`,
-   :tacn:`destruct` generates no induction hypothesis.
+   inductive or co-inductive type selected by :n:`@induction_arg`.  The selected
+   subterm, after possibly doing an :tacn:`intros`, must have
+   an inductive or co-inductive type.  Unlike :tacn:`induction`,
+   :n:`destruct` generates no induction hypothesis.
 
-   The :n:`@one_term` may contain holes that are denoted by “_”. In this case,
-   the tactic selects the first subterm that matches the pattern and performs
-   case analysis using that subterm.
+   In each new subgoal, the tactic replaces the selected subterm with the associated
+   constructor (applied to its arguments).
 
    :n:`{+, @induction_clause }`
      Giving multiple :n:`@induction_clause`\s is equivalent to applying :n:`destruct`
      serially on each :n:`@induction_clause`.
 
+   :n:`@induction_arg`
+     + If :n:`@one_term` (in :n:`@one_term_with_bindings`)
+       is an identifier :n:`@ident` denoting a quantified variable of the
+       goal, then :n:`destruct @ident` behaves like :tacn:`intros` :n:`until
+       @ident; destruct @ident`. :n:`@one_term` must have an inductive type.
+       If :n:`@ident` is no longer dependent in the
+       goal after application of :n:`destruct`, it is erased. To avoid erasure,
+       use parentheses, as in :n:`destruct (@ident)`.
+     + :n:`@one_term` may contain holes that are denoted by “_”. In this case,
+       the tactic selects the first subterm that matches the pattern and performs
+       case analysis using that subterm.
+     + If :n:`@induction_arg` is a :n:`@natural`, then :n:`destruct @natural` behaves like
+       :n:`intros until @natural` followed by :n:`destruct` applied to the last
+       introduced hypothesis.
+
    :n:`@natural`
      Equivalent to :tacn:`intros` :n:`until @natural` followed by `destruct` applied to
      the last introduced hypothesis.
```

Comment:
Note that this suggestion encompasses several changes:

- the note on `ident` being no longer dependent in the goal is not specific to the case where `ident` is a quantified variable of the goal
- `one_term` must have an inductive type was redundant with what is already written above and didn't belong in the middle of a bullet-point about `ident`
- the syntax case on `natural` was redundant with the corresponding bullet-point.

```suggestion
   :n:`@induction_arg`
     + If :n:`@one_term` (in :n:`@one_term_with_bindings`)
       is an identifier :n:`@ident`:
       + If :n:`@ident` denotes a quantified variable of the
         goal, then :n:`destruct @ident` behaves like :tacn:`intros` :n:`until
         @ident; destruct @ident`.
       + If :n:`@ident` is no longer dependent in the
         goal after application of :n:`destruct`, it is erased. To avoid erasure,
         use parentheses, as in :n:`destruct (@ident)`.
     + :n:`@one_term` may contain holes that are denoted by “_”. In this case,
       the tactic selects the first subterm that matches the pattern and performs
       case analysis using that subterm.
     + If :n:`@induction_arg` is a :n:`@natural`, then :n:`destruct @natural` behaves like
       :n:`intros until @natural` followed by :n:`destruct` applied to the last
       introduced hypothesis.
```